docs/content/Caching/Using-Pre-Aggregations.md

```diff
@@ -25,7 +25,30 @@ Refresh strategy can be customized by setting the
 pre-aggregation.
 
 The default value of `refreshKey` is `every: '1 hour'`. It can be redefined
-either by providing SQL:
+either by overriding the default value of the [`every`
+property][ref-schema-ref-preaggs-refresh-key-every]:
+
+```javascript
+cube(`Orders`, {
+  // ...
+
+  preAggregations: {
+    amountByCreated: {
+      type: `rollup`,
+      measures: [amount],
+      timeDimension: createdAt,
+      granularity: `month`,
+      refreshKey: {
+        every: `12 hour`,
+      },
+    },
+  },
+});
+```
+
+Or by providing a [`sql` property][ref-schema-ref-preaggs-refresh-key-sql]
+instead, and leaving [`every` unchanged from its' default
```

Comment:
in-memory and pre-aggs have different default.

Reply:
I'll confirm with @ovr and update the PR accordingly